prepare-json.js

```diff
@@ -50,6 +53,33 @@ function getNextEvents(member) {
   return member;
 }
 
+function getPreviousEvents(member) {
+
+  const eventEmitter = Events.getEventsEmitter()
+
+  eventEmitter.on('getPrevFromSourceInit', (source, options) => {
+    console.log(`    · Getting previous events json for ${colors.green(options.member.name)} from ${colors.underline(source.type)}`);
+  })
+
+  eventEmitter.on('getPrevFromSourceCompleted', (previousEvents, options) => {
+    if (previousEvents.length === 0) {
+      console.log(`        ${colors.yellow(`No previous events found`)}`)
+    } else {
+      previousEvents = !Array.isArray(previousEvents) ? [previousEvents] : previousEvents
```

Comment:
I would do the condition in the other way around. Avoiding the use of negated conditions 😄